manifests/n/NetSarangComputer/PortX/2.1.10/NetSarangComputer.PortX.installer.yaml

```diff
@@ -0,0 +1,17 @@
+# Created with YamlCreate.ps1 v2.1.0 $debug=AUSU.5-1-19041-1320
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.1.0.schema.json
+
+PackageIdentifier: NetSarangComputer.PortX
+PackageVersion: 2.1.10
+MinimumOSVersion: 10.0.0.0
+InstallerType: nullsoft
+Scope: user
+InstallerSwitches:
+  Silent: /S
+  SilentWithProgress: /S
```

Comment:
```suggestion
```
These switches are already passed to the Installer as InstallerType is set to `nullsoft`.

https://github.com/microsoft/winget-cli/blob/c3b8d5412d90659277ab3ca6e324226b40c4679a/src/AppInstallerCommonCore/Manifest/ManifestCommon.cpp#L579